model/Vocabulary.php

```diff
@@ -615,19 +615,15 @@ public function verifyVocabularyLanguage($lang)
 
     /**
      * Returns a list of recently changed or entirely new concepts.
+     * @param string $prop the property uri pointing to timestamps, eg. 'dc:modified'
      * @param string $clang content language for the labels
-     * @param string $lang UI language for the dates
+     * @param int $offset starting index offset
+     * @param int $limit maximum number of concepts to return
      * @return Array
      */
-    public function getChangeList($prop, $clang, $lang, $offset)
+    public function getChangeList($prop, $clang, $offset, $limit)
     {
-      $changelist = $this->getSparql()->queryChangeList($clang, $offset, $prop);
-      $bydate = array();
-      foreach($changelist as $concept) {
-        $concept['datestring'] = Punic\Calendar::formatDate($concept['date'], 'medium', $lang);
-        $bydate[Punic\Calendar::getMonthName($concept['date'], 'wide', $lang, true) . Punic\Calendar::format($concept['date'], ' y', $lang) ][strtolower($concept['prefLabel'])] = $concept;
-      }
-      return $bydate;
+      return $this->getSparql()->queryChangeList($clang, $offset, $prop, $limit);
```

Comment:
I can't help noticing that the parameter order for queryChangeList is completely different from getChangeList for no obvious reason. I suggest changing the parameter order to `$prop, $clang, $offset, $limit` which seems logical (e.g. offset and limit are next to each other)

Reply:
The inconsistency of the parameter order was running pretty deep in the method call stack. Fixed it, thanks for noticing it!